pkg/database/database.go

```diff
@@ -529,6 +529,13 @@ func GetEmailSubscribers(conn *sql.DB) ([]EmailSubscriber, error) {
 	return res, nil
 }
 
+func CountEmailSubscribers(conn *sql.DB) (int, error) {
+	row := conn.QueryRow(`SELECT count(*) FROM email_subscribers WHERE confirmed_at IS NOT NULL`)
```

Comment:
maybe once we refactor the database to use repository it will be easier to add a caching layer. I will open up a ticket to cache this in memory using go big cache package. We are already using this for a few other items like pinned job count and other stuff :)

Reply:
Yes, makes sense to do that after the refactoring, the server is too cluttered already